examples/python_unet/unet_inference.ipynb

```diff
@@ -0,0 +1,294 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "cd7a3990",
+   "metadata": {},
+   "source": [
+    "## Import MIGraphX Python Library"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "3930d7b8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import migraphx\n",
+    "from PIL import Image\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import torch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b350c333",
+   "metadata": {},
+   "source": [
+    "## Fetch U-NET ONNX Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "02a7b7de",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2021-07-21 20:44:41--  https://github.com/cagery/unet-onnx/raw/main/unet_13_256.onnx\n",
+      "Resolving github.com (github.com)... 140.82.113.4\n",
+      "Connecting to github.com (github.com)|140.82.113.4|:443... connected.\n",
+      "HTTP request sent, awaiting response... 302 Found\n",
+      "Location: https://media.githubusercontent.com/media/cagery/unet-onnx/main/unet_13_256.onnx [following]\n",
+      "--2021-07-21 20:44:42--  https://media.githubusercontent.com/media/cagery/unet-onnx/main/unet_13_256.onnx\n",
+      "Resolving media.githubusercontent.com (media.githubusercontent.com)... 185.199.108.133, 185.199.110.133, 185.199.109.133, ...\n",
+      "Connecting to media.githubusercontent.com (media.githubusercontent.com)|185.199.108.133|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 69052820 (66M) [application/octet-stream]\n",
+      "Saving to: ‘unet_13_256.onnx’\n",
+      "\n",
+      "unet_13_256.onnx    100%[===================>]  65.85M  25.8MB/s    in 2.5s    \n",
+      "\n",
+      "2021-07-21 20:44:45 (25.8 MB/s) - ‘unet_13_256.onnx’ saved [69052820/69052820]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget -nc https://github.com/cagery/unet-onnx/raw/main/unet_13_256.onnx"
```

Comment:
The model downloaded here causes the kernel to restart when I try to parse it. Trying to read the model from the driver fails with the only output being `Killed`. Cagri was unable to reproduce the error; curious to see if you encounter the same issue or not @kahmed10 .

Reply:
The only difference I can think of is that GPUs are different. 
On ROCm versions 4.1 and 4.2, I still do not see the error. 
I, although, see `MIOpen(HIP): Error [BaseGetSolution] result.out_pix_tile1 == 0 || result.out_pix_tile0 == 0` but this does not cause any failures. Maybe this error causes different behavior on your GPU and causing it to kill?

Reply:
I have ROCm 4.2, not seeing the error either.

Reply:
I have provided another unet onnx file that uses transposed convolutions instead of bilinear to @turneram, he did not see the issue with transposed conv. Might be worth looking into it as a separate issue for why bilinear upsampling is causing issues in his system.